docs/scorecard.rst

```diff
@@ -1,136 +1,188 @@
-
-**<field>** describes an individual row or row group in the scorecard table. Attributes: 
-
-**name:** database field name 
-
-**<val>** describes a value of the field. Field can have more than one value element. Attributes: 
-
-**name:** database field name 
-
-**label:** the label that shows up on the image.
-
-**<field>** next level database field name. Field can have more than one field element.
-
-**<columns>** describes an individual column in the scorecard table. Contains one or more nested **<field>** elements 
-
-**<field>** describes an individual column or column group in the scorecard table. Attributes: 
-
-**name:** database field name 
-
-**<val>** describes a value of the field. Field can have more than one value element. Attributes: 
-
-**name:** database field name 
-
-**label:** the label that shows up on the image 
-
-**<field>** next level database field name. Field can have more than one field element.
-
-**<agg_stat>** TRUE(bootstrapping) or FALSE(student-T/normal), indicating whether or not to create a scorecard using aggregated statistics. The default value is TRUE (use bootstrapping ) 
-
-**<boot_repl>** number of bootstrapping replications, use 1 for no bootstrapping. Used when **<agg_stat>** is TRUE.
-
-**<boot_random_seed>** value of Random.seed for bootstrapping to get a reproducible random results. Used when **<agg_stat>** is TRUE.
-
-**<plot_stat>** name of the aggregation statistic: median or mean. The default value is median. 
-
-**<tmpl>** 
+How-To
+-------
 
-**<data_file>** name of the R data.frame data file, inside of the **<r_work>** folder specified above.
+The scorecard module is used to generate a scorecard PNG and HTML images using data in the METviewer database. The usage statement:
 
-**<plot_file>** name of the output image file, inside of the **<plots>** folder specified above.
+**Usage:**
 
-**<title>** title of the scorecard displayed on the image. Default value: Verification Scorecard 
+.. code-block:: none
 
-Examples
---------
+  mv_scorecard.sh scorecard_example.xml
+
+where scorecard_example.xml is the XML that contains all of your scorecard specifications.
+
+The scorecard_example.xml passed to the scorecard module contains information about the scorecard that will be generated. 
+
+.. literalinclude:: scorecard_example.xml
+
+Here is an explanation for the sections:
+
+**<plot_spec>:** This is the top level tag. It stands for "plot specifications". Everything is defined in this tag.
+
+**<connection>:** Please reference the "Common XML Structures" documentation. **NOTE: This is in Tatiana's HTML documentation we'll need to incorporate**
+
+**<rscript>:** Specifies the Rscript instance to use, for example /usr/local/bin/Rscript.
+
+**<folders>:** A structure that specifies the input and output file system folders.
+        
+        **<r_tmpl>:** The folder that contains the R templates used by METviewer, typically *R_tmpl/* from the metviewer CVS source tree.
```

Comment:
```suggestion
        **<r_tmpl>:** The folder that contains the R templates used by METviewer, typically *R_tmpl/* from the METviewer CVS source tree.
```
Lindsay, should METviewer be capitalized here? I am not sure, but thought I should ask.